app/assets/stylesheets/dm/pages/_partials/_header.scss

```diff
@@ -1,5 +1,6 @@
 @import "../../variables/colors";
 @import "../../../dm_uswds/theme/uswds-theme-components";
+$underline: none !important;
```

Comment:
this is confusing.
are we changing this to no underline? also, it is specifically the `underline`? `text-decoration` can be other things.
if so, this should be ```$decoration-none``` or something like that

Reply:
also, this can be moved to a variables file so other files can use it
be sure to include the variable file in this file

Reply:
Updated! Thank you for pointing these out.